README.md

```diff
@@ -26,7 +26,7 @@ You need Linux or macOS. On Windows, we recommend installing the Windows Subsyst
 - (Recommended) ASAN (Address Sanitizer). We use this to catch memory errors early. `apt install libasan`. (If you wish to use LOST without the address sanitizer, build it with `make LOST_DISABLE_ASAN=1`, or set `export LOST_DISABLE_ASAN=1` in your `~/.bashrc` file to disable it every time you build).
 - Git. On Debian, `apt install git`
 - Cairo, to read and write PNGs as well as draw on them for debugging and demo purposes. You won't
-  need this on your CubeSat. On Debian, `apt install libcairo2-dev`. Elsewhere, follow the
+  need this on your CubeSat. On Debian, `sudo apt-get install libcairo2-dev`. Elsewhere, follow the
```

Comment:
Debian recommends using `apt` instead of `apt-get` when used interactively at the command-line; apt-get is only meant for use by automation.